src/ResourceManager/Websites/Commands.Websites/Cmdlets/DeploymentSlots/SetAzureWebAppSlot.cs

```diff
@@ -80,8 +80,12 @@ public class SetAzureWebAppSlotCmdlet : WebAppSlotBaseCmdlet
         [ValidateNotNullOrEmpty]
         public bool Use32BitWorkerProcess { get; set; }
 
-        [Parameter(Mandatory = false, HelpMessage = "Destination slot name for auto swap")]        
+        [Parameter(Mandatory = false, HelpMessage = "Destination slot name for auto swap")]
+
```

Comment:
@LukaszStem I still see there are ~15 positional parameters. Any reasons for having so many positional parameters?
Also why not make all your parameters have a default value, rather than having it assigned null during construction of SiteConfig.
Nit: If you decide not to have default value for each of your parameter and initialize in the SiteConfig constructor, can you indent it so that readability is much better.